.github/workflows/build_packages.yml

```diff
@@ -0,0 +1,72 @@
+name: "Test rippled packaging"
+on:
+  push:
+    branches: ['master', 'release', 'develop', 'develop-next', '*-rc*', '*-b*']
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  build_packages:
+      name: Build packages
+      runs-on: [self-hosted, Linux]
+      strategy:
+        fail-fast: false
+        matrix:
+          type:
+            - suffix: deb
+              image: rippleci/rippled-dpkg-builder:2020-02-10
+              script: dpkg
+            - suffix: rpm
+              image: rippleci/rippled-rpm-builder:2020-02-10
+              script: rpm
+      container:
+        image: ${{ matrix.type.image }}
+
+      steps:
+        - name: Checkout
+          uses: actions/checkout@v3
```

Comment:
This step is unnecessary because it is already in the `package` action, right?

Reply:
Checkout can be removed from the action since it's in the workflow.